### ISSUE TITLE
Compilation error at V8.1.0

### ISSUE THREAD

**Issue by coolnk2001** [fknop/angular-pipes#71]
**I'm submitting a ...**  (check one with "x")

```
[ x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
```

**Current behavior**

Update to V8.1.0 cause compilation error 
Module not found: Error: Can't resolve 'angular-pipes'

Revert back to V8.0.0 problem gone

**Expected behavior**

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**

**Comment by fknop** [comment]
Indeed, fixing this right now thank you.

**Comment by fknop** [comment]
Should be fixed with 8.1.1.
